src/test/java/com/prgrms/commandLineApplication/repository/JdbcCustomerRepositoryTest.java

```diff
@@ -0,0 +1,104 @@
+package com.prgrms.commandLineApplication.repository;
+
+import com.prgrms.commandLineApplication.customer.Customer;
+import com.prgrms.commandLineApplication.repository.customer.JdbcCustomerRepository;
+import org.junit.jupiter.api.*;
+import org.assertj.core.api.Assertions;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.Rollback;
+
+import java.util.NoSuchElementException;
+import java.util.UUID;
+
+@SpringBootTest
```

Comment:
레포지토리를 테스트하는데 모든 빈들을 등록할 필요는 없을 것 같아요.

```suggestion
@Import(JdbcCustomerRepository.class)
@JdbcTest
```

Reply:
b54a3728